whc-shared/src/test/java/com/ibm/whc/deid/shared/pojo/masking/PhoneMaskingProviderConfigTest.java

```diff
@@ -36,5 +40,39 @@ public void testValidate() throws Exception {
     }
     config.setInvNdigitsReplaceWith("1");
     config.validate();
+
+    List<String> patterns = new ArrayList<>();
+    config.setPhoneRegexPatterns(patterns);
+    patterns.add("[0-9");
+    try {
+      config.validate();
+      fail("expected exception");
+    } catch (InvalidMaskingConfigurationException e) {
+      assertTrue(
+          e.getMessage().startsWith("pattern at offset 0 in `phoneRegexPatterns` is not valid: "));

```

Comment:
Kinda shows what I was talking about above. Here we could just check that the error has WPH1234E and then know for sure what tokens to expect. Useful in some cases for the tool side or troubleshooting guides

Reply:
Discussed with Denis.  Will address in a follow-up change.